packages/console/src/pages/Organizations/Guide/CreatePermissions/Introduction/Diagram.tsx

```diff
@@ -0,0 +1,493 @@
+/* eslint-disable max-lines */
+import * as styles from './index.module.scss';
+
+function Diagram() {
```

Comment:
this large file can be hard to maintain. can we figure out a more structured way like divide into more components?

Reply:
This will be improved later this week. I had a better plan when I was implementing the interaction on the diagram.
Will drive the refactor in this one: https://linear.app/silverhand/issue/LOG-7374/make-the-introduction-diagram-interactive